core/utility.ml

```diff
@@ -1458,5 +1458,3 @@ struct
         (sequence xs) >>= fun xs ->
         Lwt.return (x :: xs)
 end
-
```

Comment:
Please avoid changing trailing whitespace for no reason

Reply:
Actually, maybe it is better to remove both trailing whitespace lines, these probably should have been removed already.  In any case, there is no point in just removing one.

Reply:
Is there a code style I can check?

Reply:
Supposedly the script `src/links/tools/rules/trailing_whitespace.sh` checks for trailing whitespace.  But it doesn't seem to object to newlines at the end of a file - I keep tripping over this because my editor is configured to remove them from ocaml files.

Here I'm just reacting to the fact that utility.ml seems to have been changed for no reason possibly because your editor is doing the same thing.  Probably we should standardize on removing spurious newlines from the ends of files (or at least not complain reflexively about them being removed like I have).